libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime/Settings/SkillConfigurationEntry.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime.Settings
+{
+    /// <summary>
+    /// Definition of a skill entry in the appsettings.json file.
+    /// </summary>
+    public class SkillConfigurationEntry
```

Comment:
>SkillConfigurationEntry [](http://example.com/codeflow?start=17&length=23)

This object would also need to store the skill's messaging endpoint: https://<myskillapp>.azurewebsites.net/api/messages.

Reply:
We don't need the endpoint in this specific case as we are only validating that an incoming call is from the AppId that we had configured as an outbound skill we call. The url is pulled out in other code outside the validation side to get the url for outbound skills calls.